data/json/monsters/reptile_amphibian.json

```
@@ -195,4 +195,49 @@
     "path_settings": { "max_dist": 5 },
     "flags": [ "SEES", "SMELLS", "WARM", "VENOM", "SWIMS" ]
   }
+{
+    "id": "mon_turtle_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant turtle" },
+    "description": "A horrifically large and ulcerous snapping turtle. In the air above its primordial shell it interweaves its unnaturally elongated and flexible neck and tail with unnervingly elegant complexity.",
+    "default_faction": "mutant",
+    "bodytype": "gator",
+    "species": [ "AMPHIBIAN" ],
+    "volume": "92500 ml",
+    "weight": "120 kg",
+    "hp": 150,
+    "speed": 10,
+    "attack_cost": 10,
+    "material": [ "flesh", "bone" ],
+    "symbol": "T",
+    "color": "light_green",
+    "aggression": 50,
+    "morale": 100,
+    "melee_skill": 10,
+    "melee_dice": 1,
+    "melee_dice_sides": 20,
+    "melee_cut": 10,
+    "dodge": 1,
+    "armor_bash": 30,
+    "armor_cut": 30,
+    "armor_bullet": 30,
+    "path_settings": { "max_dist": 5 },
+     "special_attacks": [
+      {
+        "type": "bite",
+        "cooldown": 10,
+        "accuracy": 6,
+        "damage_max_instance": [
+          { "damage_type": "stab", "amount": 9, "armor_multiplier": 0.5 },
+          { "damage_type": "bash", "amount": 27, "armor_multiplier": 0.5 }
+        ],
+        "effects": [ { "id": "grabbed", "duration": 1000, "bp": "torso" } ],
+        "infection_chance": 50
+      }
+],
+    "anger_triggers": [ "PLAYER_CLOSE" ],
```

```suggestion
    "anger_triggers": [ "PLAYER_CLOSE", "HOSTILE_SEEN" ],
```
Together with the aggression changes it should mean they only get ticked off when they can do something about it.